core/src/main/scala/kafka/log/TimeIndex.scala

```diff
@@ -51,7 +50,7 @@ import org.apache.kafka.common.record.RecordBatch
  */
 // Avoid shadowing mutable file in AbstractIndex
 class TimeIndex(_file: File, baseOffset: Long, maxIndexSize: Int = -1, writable: Boolean = true)
-    extends AbstractIndex[Long, Long](_file, baseOffset, maxIndexSize, writable) with Logging {
+    extends AbstractIndex[Long, Long](_file, baseOffset, maxIndexSize, writable) {
```

Comment:
This class also should provide the Logger, Right?

Reply:
There are no log statements in this class.

Reply:
There is one statement https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/log/TimeIndex.scala#L129

Reply:
Good catch! CoreUtils was exposing its logger, so I fixed both in a new commit.